snow/engine/snowman/syncer/state_syncer.go

```diff
@@ -248,12 +253,21 @@ func (ss *stateSyncer) AcceptedStateSummary(ctx context.Context, nodeID ids.Node
 		newWeight, err := math.Add64(weight, ws.weight)
 		if err != nil {
 			ss.Ctx.Log.Error("failed to calculate the Accepted votes",
+				zap.Stringer("summaryID", summaryID),
+				zap.Uint64("height", ws.summary.Height()),
 				zap.Uint64("weight", weight),
 				zap.Uint64("previousWeight", ws.weight),
 				zap.Error(err),
 			)
 			newWeight = stdmath.MaxUint64
 		}
+
+		ss.Ctx.Log.Debug("adding weight to summary",
+			zap.Stringer("summaryID", summaryID),
+			zap.Uint64("height", ws.summary.Height()),
+			zap.Uint64("previousWeight", ws.weight),
+			zap.Uint64("newWeight", newWeight),
```

Comment:
nit: should we use `currentWeight` instead of `newWeight`. Same for log pre increment